### DISCUSSION TITLE
is there any class for input checkbox checked mark ?

### DISCUSSION THREAD

**Discussion by JaimeenMakavana** [tailwindlabs/tailwindcss#12952] [Help]
I can able to style background but not able to style checked mark color ?     <input type="checkbox" name="" id="" class="checked:accent-red-600 size-28">

**Comment by alex-krasikau** [comment]
Hey @AtharvaMistry!

There's an `accent-color` property that controls the background of the input elements, but as you already mentioned, it updates the background and doesn't let you customize the check icon.

To achieve this, you will need to build a completely custom checkbox. This approach will allow you to control the checkbox background, check color, and introduce any kind of extra customization, such as the appearance animation.

Here's the simplified solution:

```html
<div>
  <label>
    <input type="checkbox" class="sr-only [&:checked+span>svg]:block [&:checked+span]:bg-blue-400" />
    <span class="inline-block h-6 w-6 cursor-pointer rounded-md border">
      <svg class="hidden">
        <path d="..." ></path> <!-- Check Icon -->
      </svg>
    </span>

    <span>Checkbox name</span>
  </label>
</div>
```

Here you can find the complete solution - [https://play.tailwindcss.com/UkusWA9pEK](https://play.tailwindcss.com/UkusWA9pEK)
